scripts/get_release_from_bundle_source.py

```diff
@@ -0,0 +1,33 @@
+# Get bundle test path for specific release
+import os
+import re
+import sys
+
+def get_release_from_bundle_source() -> None:
+    if len(sys.argv) <= 1:
+        raise Exception("No bundle source given as input.")
+
+    bundle_source = sys.argv[1]
+    # Bundle source input should be `--channel <channel_name>` or `--file <bundle_file>.yaml``
+    # e.g. --channel=1.8/stable or --file=releases/1.8/stable/kubeflow/bundle.yaml
+    bundle_source_starts_with_channel = re.search("^--channel", bundle_source)
+    bundle_source_starts_with_file = re.search("^--file", bundle_source)
+
+    try:
+        if bundle_source_starts_with_channel:
+            substrings = bundle_source.split('=')
```

Comment:
what if the input is `--channel <channel_name>` without the `=`, would this work?